workflows/RNA-seq/RNA-seq

```diff
@@ -74,6 +74,13 @@ def parse_args(defaults={"verbose":None,"configfile":None,"max_jobs":None,"snake
                         help="configuration file: config.yaml. In absence of a config file, the default options from the "
                         "workflows would be selected from /workflows/<workflow_name>/defaults.yaml (default: '%(default)s')",
                         default=defaults["configfile"])
+    
+    general.add_argument("--cluster_configfile",
+                        dest="cluster_configfile",
+                        help="configuration file for cluster usage. In absence, the default options "
+                        "from shared/cluster.yaml and [workflow]/cluster.yaml would be selected (default: '%(default)s')",
+                        default=None)
+    
```

Comment:
> Configuration file for cluster usage. If not specified, `shared/cluster.yaml` and `[workflow]/cluster.yaml` will be used.

You can then remove the `(default: ...)` stuff.

Reply:
done